_posts/samus_returns/2024-04-22-export-romfs.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "How to extract your Metroid: Samus Returns RomFS"
+tags: samus_returns
+---
+You can either extract your RomFS using Godmode9 from your 3DS, or using Citra from an installation file (.3ds or .cia). We'll show you how to do it using both methods.
+
+## Citra
+
+First, download and setup Citra. Once you've set it up, navigate to **File -> Install CIA...** to add the directory with your game image or simply double click on **Add New Game Directory** in the main window. Your game should be shown in Citra now.
+
+![](/assets/guides/romfs_msr/citra1.png)
```

Comment:
please fill in teh alt text

Reply:
Funnily enough, these are empty for Dread 😛

Reply:
yeah, see #34